src/protocols/mcsp/tests/TestMessageCounterManager.cpp

```diff
@@ -86,7 +83,7 @@ class TestExchangeMgr : public SecureSessionMgrDelegate
     {
         NL_TEST_ASSERT(mSuite, header.GetSourceNodeId() == Optional<NodeId>::Value(kSourceNodeId));
         NL_TEST_ASSERT(mSuite, header.GetDestinationNodeId() == Optional<NodeId>::Value(kDestinationNodeId));
-        NL_TEST_ASSERT(mSuite, msgBuf->DataLength() == kMsgCounterChallengeSize);
+        NL_TEST_ASSERT(mSuite, msgBuf->DataLength() == 8);
```

Comment:
please add a constant for 8 - what is 8  and why is it this specific size?

Reply:
done